src/Main.java

```diff
@@ -1,5 +1,54 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Привет, Мир!");
+
+        System.out.println("Task 1");
+        checkLeapYear(2025);
+
+        System.out.println("Task 2");
+        recommendAppVersion(0, 2014);
+        recommendAppVersion(1, 2016);
+
+        System.out.println("Task 3");
+        int deliveryDistance = 95;
+        System.out.println("Потребуется дней: " + calculateDeliveryDays(deliveryDistance));
+    }
+
+    public static void checkLeapYear(int year) {
+
+        boolean isLeapyear = (year % 4 == 0 && year % 100 != 0 || year % 400 == 0);
```

Comment:
```suggestion
        boolean isLeapyear = year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
```
скобки не нужны

Reply:
также переменные неправильно называть на is, методы так должны начинаться, а переменные без is. А вот camelCase надо соблюдать
```suggestion
        boolean leapYear = (year % 4 == 0 && year % 100 != 0 || year % 400 == 0);
```